package/python-kiwi-spec-template

```diff
@@ -364,6 +344,102 @@ Group:          %{sysgroup}
 
+%package -n kiwi-host-setup-for-wsl
+Summary:        KIWI - host requirements for building wsl images
+Group:          System/Management
+Provides:       kiwi-image:wsl
+Requires:       fb-util-for-appx
+
+%description -n kiwi-host-setup-for-wsl
+Host setup helper to pull in all packages required/useful on
+the build host to build wsl/appx images
+
+%package -n kiwi-host-setup-for-docker
+Summary:        KIWI - host requirements for docker images
+Group:          System/Management
+Provides:       kiwi-image:docker
+Requires:       skopeo
+Requires:       umoci
+
+%description -n kiwi-host-setup-for-docker
+Host setup helper to pull in all packages required/useful on
+the build host to build docker images
+
+%package -n kiwi-host-setup-for-live-media
+Summary:        KIWI - host requirements for live iso images
+Group:          System/Management
+Provides:       kiwi-image:iso
+Requires:       checkmedia
+Requires:       dosfstools
+Requires:       xorriso
+%ifarch %{ix86} x86_64
+Requires:       syslinux
+Requires:       gfxboot
+%endif
+
+%description -n kiwi-host-setup-for-live-media
+Host setup helper to pull in all packages required/useful on
+the build host to build live iso images.
+
+%package -n kiwi-host-setup-for-install-media
+Summary:        KIWI - host requirements for install iso images
+Group:          System/Management
+Requires:       kiwi-host-setup-for-live
+
+%description -n kiwi-host-setup-for-install-media
+Host setup helper to pull in all packages required/useful on
+the build host to build install iso images
+
+%package -n kiwi-host-setup-for-filesystems
+Summary:        KIWI - host requirements for filesystems
+Group:          System/Management
+Provides:       kiwi-image:pxe
+Provides:       kiwi-image:kis
+Provides:       kiwi-filesystem:btrfs
+Provides:       kiwi-filesystem:ext3
+Provides:       kiwi-filesystem:ext4
+Provides:       kiwi-filesystem:squashfs
+Provides:       kiwi-filesystem:xfs
+Requires:       e2fsprogs
+Requires:       xfsprogs
+%if 0%{?fedora} || 0%{?rhel}
+Requires:       btrfs-progs
+Requires:       squashfs-tools
+%else
+Requires:       btrfsprogs
+Requires:       squashfs
+%endif
+
+%description -n kiwi-host-setup-for-filesystems
+Host setup helper to pull in all packages required/useful on
+the build host to build filesystem images
+
+%package -n kiwi-host-setup-for-disk
+Summary:        KIWI - host requirements for disk images
+Group:          System/Management
+Provides:       kiwi-image:oem
+Provides:       kiwi-image:vmx
+Requires:       kiwi-host-setup-for-filesystems
+
+%description -n kiwi-host-setup-for-disk
+Host setup helper to pull in all packages required/useful on
+the build host to build disk images
+
+%package -n kiwi-host-setup-for-validation
+Summary:        KIWI - host requirements for handling image descriptions better
+Group:          System/Management
+Requires:       jing
+%if 0%{?suse_version}
+Requires:       Requires:       python%{python3_pkgversion}-solv
+%endif
```

Comment:
Why do we require the Python bindings for libsolv? Also, `Requires: Requires:` is not valid.

Reply:
libsolv is needed if you call:

kiwi-ng image info --description foo --resolve-package-list

very handy for debugging and in case of errors. That's why I put it into the kiwi-host-setup-for-validation sub package

Reply:
There is the `kiwi-ng image info --resolve-package-list` command that resolves the packages that are required to build the image without actually build it. This is helpful for debugging the required list of packages without actually having to launch a build for some smoke test verification.